src/utils/data/record.ts

```diff
@@ -1,351 +0,0 @@
-import * as Sentry from '@sentry/node';
```

Comment:
why it's so helpful. 😄

Reply:
?

Reply:
Ah, you mean why did I delete the file?

I thought it wasn't used and it contains a lot of unused code now, were you using it?

Reply:
@samuelcastro Re-implemented at https://github.com/UnlyEd/next-right-now/commit/0fd1aae03a283cf79e810467d07f3ec03a0f5508 (and cleaned)

Reply:
yeah so, I saw this utility some time ago and it think it's really helpful, glad to brought it back. :)